ansibledriver/ibm_cp4na_log_message.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/python
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+# This custom module is for printing some specified logs
+
+from ansible.module_utils.basic import AnsibleModule
+
+def run_module():
+    # defining available arguments/parameters a user can pass to the module
+    module_args = dict(
+        message_direction=dict(type='str', required=True, choices=['sent', 'received']),
+        external_request_id=dict(type='str', required=True),
+        message_data=dict(type='str'),
+        content_type=dict(type='str'),
+        protocol=dict(type='str', required=True),
+        protocol_metadata=dict(type='dict', default={})
+    )
+
+    result = dict()
+
+    module = AnsibleModule(
+        argument_spec=module_args,
+        supports_check_mode=True,
+        required_together=[['message_data', 'content_type']]
+    )
+
+    if(module.check_mode):
+        module.exit_json(**result)
+
+    message_direction = module.params['message_direction']
+    external_request_id = module.params['external_request_id'] 
+    message_data = module.params['message_data']
+    content_type = module.params['content_type']
+    protocol = module.params['protocol']
+    protocol_metadata = module.params['protocol_metadata']
+
+    if(protocol is None):
+        module.fail_json("Please provide a non empty protocol")
+    if(external_request_id is None):
+        module.fail_json("Please provide a non empty external_request_id")
```

Comment:
`message_direction` and `message_type` (if we add it) should be mandatory as well